src/RestApi/StoreApi/Schemas/ProductSchema.php

```diff
@@ -31,64 +31,64 @@ class ProductSchema extends AbstractSchema {
 	 */
 	protected function get_properties() {
 		return [
-			'id'             => array(
+			'id'            => array(
```

Comment:
It may be best to split the REST API changes out of this PR to review separately, since those were just missing data.

Reply:
Can do 👍

Reply:
done in #1116